Content.Server/Botany/Components/PlantAnalyzerComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Botany.Components;
+
+/// <summary>
+///    After scanning, retrieves the target Uid to use with its related UI.
+/// </summary>
+[RegisterComponent]
+public sealed partial class PlantAnalyzerComponent : Component
+{
+    /// <summary>
+    ///     How long it takes to scan.
+    /// </summary>
+    [DataField("scanDelay")]
+    public float ScanDelay = 0.8f;
```

Comment:
I think I would be okay with this if the default `ScanDelay` was increased to something in the order of 30 seconds.

Reply:
As I said on discord, I am working on an updated branch of this and made an advanced scanning function that takes 30 seconds to display all information, while the basic scan shows only an extra two traits that weren't player visible before. 

I am not married to 0.8 because I just copied and pasted that, I would suggest something like 2 or 3 for now. This is a new feature, people should be encouraged to use and test it, then we can always crank it up afterward if necessary.
 
But 30 seconds for the basic scan one would be way too much. Spending more than 5 minutes to scan all trays a single time is absurd.

Reply:
20 for basic scan and 40 for advanced, also advanced uses double the cell power, does that seem okay?